apps/web/src/components/molecules/textArea/TextArea.module.scss

```diff
@@ -0,0 +1,33 @@
+.areaWrapper { 
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 8px;
+}
+
+.textArea {
+    background-color:#E6E6E6;
+    color: #7A7A7A;
```

Comment:
a to nie były kolory ze zmiennych? Ew. czy background i black tu nie będą pasować?

Reply:
wszystkie kolory i teksty bralem z figmy. Poprawilem juz na te z colors